packages/json-api/src/-private/builders/save-record.ts

```diff
@@ -52,6 +52,48 @@ export function deleteRecord(record: unknown, options: ConstrainedRequestOptions
   };
 }
 
+/**
+ * Builds request options to create new record for resources,
+ * configured for the url, method and header expectations of most JSON:API APIs.
+ *
+ * **Basic Usage**
+ *
+ * ```ts
+ * import { createRecord } from '@ember-data/json-api/request';
+ *
+ * const person = this.store.createRecord('person', { name: 'Ted' });
```

Comment:
@runspired am I correct here that this is how new record will be created? 

It seems like I am missing something here... kinda feels not right with `this.store.createRecord` and `create-record` helper

Reply:
confirmed on ember-data meeting